StyleCop.Analyzers/StyleCop.Analyzers/StyleCop.Analyzers.csproj

```diff
@@ -56,36 +56,6 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>DocumentationResources.resx</DependentUpon>
     </Compile>
-    <Compile Include="DocumentationRules\DocumentationResources.de-DE.Designer.cs">
```

Comment:
:memo: These designer files weren't actually used for anything, so they could just be removed.